client/stylesheets/_toCat.scss

```diff
@@ -5355,6 +5376,10 @@ On click the images will be displayed at normal size to complete the effect
 	left:12px;
 }
 
+.optionsItems {
+    margin: 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces